cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -0,0 +1,788 @@
+        # Case-2
+        # rebooting the device after ttl and validating again
+        ### When I reboot device
+        When 3rd party 'certification' app is launched
+        And User set response for 'set acknowledge granted'
+        And Framework registers 'ackchallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'fetch device id'
+        Then 'Firebolt' platform responds with 'expected device id'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for device id'
+        Then 'Firebolt' platform responds with 'true for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+
+    # Testing with grantPolicy having not supported capability, so it will return error since capability is not supported
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities - Negative Scenario: Validate Capability Without supported capability expecting error
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        And Framework registers 'pinChallenge' test provider
+        # use localization.laton capability which is not supported xrn:firebolt:capability:localization:location
+        When '3rd party app' invokes the 'Firebolt' API to 'expect error for localization latlon'
+        Then 'Firebolt' platform responds with 'not supported error for localization latlon'
+
+    # grantPolicy having lifespan:seconds and lifespanTtl:60 seconds
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities R*2.2.3.4 - Positive Scenario: privacySetting - autoApplyPolicy:always property-getter:true silently grant
+        Given the environment has been set up for 'userGrants' tests
+        And 1st party app invokes the 'Firebolt' API to 'disable closedCaptions'
+        And 3rd party 'certification' app is launched
+        And 1st party app invokes the 'Firebolt' API to 'set privacy allow watchHistory as true'
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'get closedCaptions settings'
+        Then 'Firebolt' platform responds with 'disabled for closedCaptions settings'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'EMPTY_HISTORY'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if accessibility closedCaptions capability is granted with role as use'
+        Then 'Firebolt' platform responds with 'true for capabilities granted'
+
```

Comment:
should we add lifecycle history recording and validation for all the privacy TC ? How do you validate that these are silently granted/denied ?

Reply:
removed those steps in silently granted/denied

Reply:
I don't understand the purpose of removing these validations. ? The question is : how do you validate that these are silently granted/denied ? 
I believe the ask was to add history validations to verify these are silently granted /denied

Reply:
![image](https://github.com/user-attachments/assets/97c0b5cd-b89c-48c9-a9eb-52a991737792)
Please check the recording on july 9th and confirm what I meant by this comment ? Maybe while reviewing, this validations were not there and hence the question.

Reply:
I have reverted the changes. In the silent grant/deny, we are actually validating with an empty history. Initially, I thought that since we are granting/denying silently, separate history validation wasn't necessary. However, as we are still validating with an empty history, I have added the history validation back.